Assets/MixedRealityToolkit-Examples/Demos/StandardShader/Scripts/Editor/MixedRealityToolkit.Examples.Demos.StandardShader.Editor.asmdef

```diff
@@ -0,0 +1,12 @@
+{
+    "name": "MixedRealityToolkit.Examples.Demos.StandardShader.Editor",
```

Comment:
We've been sticking to the `Inspectors` terminology throughout the rest of the MRTK. Do you mind doing the same?

Reply:
Not at all, I'll change that!